doc/news/changes/create_changes_h.sh

```diff
@@ -29,36 +30,37 @@ if test ! -d incompatibilities -o ! -d major -o ! -d minor ; then
   exit 1
 fi
 
-OUTPUT="../changes.h.new"
 
 function process_directory
 {
-  echo "<ol>" >> ${OUTPUT}
+  echo "<ol>"
   # process all entries in the right order
   ARRAY=($(ls "$1" | sort -r))
   for f in "${ARRAY[@]}"; do
-    echo "" >> ${OUTPUT}
-    echo -n " <li>" >> ${OUTPUT}
+    echo ""
+    echo -n " <li>"
     # indent lines by one blank space
-    sed 's/^[ ]*/ /' "$1"/"$f" >> ${OUTPUT}
-    echo " </li>" >> ${OUTPUT}
+    sed 's/^[ ]*/ /' "$1"/"$f"
+    echo " </li>"
   done
-  echo "" >> ${OUTPUT}
-  echo "</ol>" >> ${OUTPUT}
+  echo ""
+  echo "</ol>"
 }
 
-cat header > ${OUTPUT}
+
+
+cat header
 
 echo INCOMPATIBILITIES
```

Comment:
Now, that we capture the console output this should just be removed. Otherwise, this also ends up in `changes.h`.